uwblueprint/planet-read
# Add translator_last_activity and reviewer_last_activity time columns

## Notion ticket link



[Add translator_last_activity and reviewer_last_activity time columns to StoryTranslation model](https://www.notion.so/uwblueprintexecs/Add-translator_last_activity-and-reviewer_last_activity-time-columns-to-StoryTranslation-model-35720c1545224b138eaf3d0c6cf35ade)



## Implementation description

- Added `translator_last_activity` and `reviewer_last_activity` time columns to database



## Steps to test

1. Upgrade database `docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db upgrade"`
2. Check that new columns `translator_last_activity` and `reviewer_last_activity` have been added to the tables `story_translations` and `story_translations_all`
    a. `docker exec -it planet-read_db_1 /bin/bash`
    b. `mysql -u root -proot`
    c. `use planet-read`
    d. `select * from story_translations;`
    e. `select * from story_translations_all;`



## What should reviewers focus on?

- Database migration is correct
- translator_last_activity and reviewer_last_activity time columns appear in database

## Checklist
